packages/suite/src/actions/wallet/selectedAccountActions.ts

```diff
@@ -53,6 +56,10 @@ const getAccountStateWithMode = (state: AppState, selectedAccount?: State) => {
         mode.push('device-unavailable');
     }
 
+    if (!isTorEnabled && selectedAccount?.account?.accountType === 'coinjoin') {
```

Comment:
no sure if this is the right place, getAccountState maybe?

Reply:
I think this is the right place but I have experience issues with this `selectedAccount?.account?.accountType` as well as `state.wallet.selectedAccount.account?.accountType` where `selectedAccount` contains its inital state `{status: 'none'}`.

When:

1. Start coinjoin account
2. Remember wallet
3. Disable Tor during discovery
4. Refresh the app (ctrl + r)
5. Go to The remembered coinjoin account that should give error with discovery since Tor is disable

In step 5 I find that right at the begging `selectedAccount` is `{status: 'none'}, and therefore this condition is not met https://github.com/trezor/trezor-suite/pull/6796/files#diff-aacbc48bf45f41a3dec12fca82dbf91f9b013a4f85332c1b05705b23b7e67097R60